src/features/registry.js

```diff
@@ -75,7 +75,12 @@ systemJSPrototype.entries = function () {
         done: key === undefined,
         value: key !== undefined && [key, ns]
       };
-    },
-    [iterator]: function() { return this }
+    }
   };
+
+  if (iterator) {
```

Comment:
I don't mind if we remove the guard. Setting the 'undefined' string in older environments isn't going to cause any issues. And yes, I do care about the bytes that much!

Reply:
👍 updated